core/packages/contracts/src/storage/CoreStorage.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023 Snowfork <hello@snowfork.com>
+pragma solidity 0.8.20;
+
+import {AgentExecutor} from "../AgentExecutor.sol";
+import {Agent} from "../Agent.sol";
+import {Channel, OperatingMode, ParaID} from "../Types.sol";
+
+library CoreStorage {
+    struct Layout {
+        // Operating mode:
+        OperatingMode mode;
+        // Message channels
+        mapping(ParaID paraID => Channel) channels;
```

Comment:
as discussed, maybe make this mapping from an agentId instead of paraId?

Reply:
I don't think there's enough time before the audit to properly spec the change and implement it.